docs/docsite/rst/roadmap/COLLECTIONS_3_0.rst

```diff
@@ -0,0 +1,45 @@
+===================
+Ansible project 3.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_11` for the most recent updates on ``ansible-base``.
+
+.. contents::
+   :local:
+
+Release schedule
+=================
+
+.. note::
+
+    Ansible its switching from its traditional versioning scheme to `semantic versioning <https://semver.org/>`_ for this release.  So the version will be 3.0.0 instead of 2.11.0.
+
+
+
+:2020-12-16: Finalize rules for net-new collections submitted for the ansible release.
+:2021-01-27: Final day for new collections to be **reviewed and approved**. They need to have been
```

Comment:
```suggestion
:2021-01-27: Final day for new collections to be **reviewed and approved**. They MUST be
```

Reply:
Heh, I struggled on this.... "need to have been" is more accurate but "MUST be" is a lot less awkward a construction.  The accuracy probably doesn't matter; we can point people who submit things late at this entry in either case.